x-pack/plugins/security_solution/public/common/containers/local_storage/use_messages_storage.test.tsx

```diff
@@ -18,48 +17,42 @@ describe('useLocalStorage', () => {
   });
 
   it('should return an empty array when there is no messages', async () => {
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseMessagesStorage>(() =>
-        useMessagesStorage()
-      );
-      await waitForNextUpdate();
-      const { getMessages } = result.current;
+    const { result } = renderHook(() => useMessagesStorage());
+    await waitFor(() => new Promise((resolve) => resolve(null)));
+    const { getMessages } = result.current;
+    act(() => {
       expect(getMessages('case')).toEqual([]);
     });
   });
 
   it('should add a message', async () => {
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseMessagesStorage>(() =>
-        useMessagesStorage()
-      );
-      await waitForNextUpdate();
-      const { getMessages, addMessage } = result.current;
+    const { result } = renderHook(() => useMessagesStorage());
+    await waitFor(() => new Promise((resolve) => resolve(null)));
+    const { getMessages, addMessage } = result.current;
+    act(() => {
       addMessage('case', 'id-1');
       expect(getMessages('case')).toEqual(['id-1']);
```

Comment:
What's the point to wrap `expect` in `act`? Shouldn't it assert the state after an action has been performed?